src/react-intl-messages.ts

```diff
@@ -1025,6 +1028,9 @@ export default {
         [AT.EDIT_TRAINDIALOG_ASYNC]: 'Editing Train Dialog',
         [AT.EDIT_APP_LIVE_TAG_ASYNC]: 'Editing Application live tag',
         [AT.EDIT_APP_EDITING_TAG_ASYNC]: 'Editing Application editing tag',
+
+        // General UX strings not fitting elsewhere
+        [FM.APP_MESSAGE_PERMDELETE]: 'Deletion is irreversible.',
```

Comment:
Still would have called this "FM.APPSLIST_MESSAGE_PERMDELETE" and put it with the other one.   (These are in the file alphabetically too)

Reply:
Folded things into inline JSX to support bold model name